code/modules/power/lighting.dm

```diff
@@ -84,6 +84,27 @@
 		to_chat(user, "<span class='danger'>This casing doesn't support power cells for backup power.</span>")
 		return
 
+/obj/structure/light_construct/attack_hand(mob/user)
+	. = ..()
+	if(.)
+		return
```

Comment:
what's the purpose of this

Reply:
I referenced the attack_hand function of something else. I read somewhere about nasty things and . = ..() so I kept it in.